src/elements/tabs/tab-group/tab-group.component.ts

```diff
@@ -111,131 +87,78 @@ class SbbTabGroupElement extends SbbHydrationMixin(LitElement) {
   protected override firstUpdated(changedProperties: PropertyValues<this>): void {
     super.firstUpdated(changedProperties);
 
-    this._tabs = this._getTabs();
-    this._tabs.forEach((tab) => this._configure(tab));
+    this.tabLabels.forEach((tabLabel) => tabLabel.linkToTab());
     this._initSelection();
     this._tabGroupResizeObserver.observe(this._tabGroupElement);
+    this._tabContentResizeObserver.observe(this._tabContentElement);
   }
 
   /**
    * Disables a tab by index.
    * @param tabIndex The index of the tab you want to disable.
    */
   public disableTab(tabIndex: number): void {
-    this._tabs[tabIndex]?.tabGroupActions?.disable();
+    this.tabLabels[tabIndex]?.disable();
   }
 
   /**
    * Enables a tab by index.
    * @param tabIndex The index of the tab you want to enable.
    */
   public enableTab(tabIndex: number): void {
-    this._tabs[tabIndex]?.tabGroupActions?.enable();
+    this.tabLabels[tabIndex]?.enable();
   }
 
   /**
    * Activates a tab by index.
    * @param tabIndex The index of the tab you want to activate.
    */
   public activateTab(tabIndex: number): void {
-    this._tabs[tabIndex]?.tabGroupActions?.select();
-  }
-
-  private _getTabs(): InterfaceSbbTabGroupTab[] {
-    return Array.from(this.children ?? []).filter((child) =>
-      /^sbb-tab-label$/u.test(child.localName),
-    ) as InterfaceSbbTabGroupTab[];
+    this.tabLabels[tabIndex]?.select();
   }
 
-  private _enabledTabs(): InterfaceSbbTabGroupTab[] {
-    return this._tabs.filter((t) => !t.hasAttribute('disabled'));
+  private _enabledTabs(): SbbTabLabelElement[] {
+    return this.tabLabels.filter((t) => !t.disabled);
   }
 
   private _updateSize(): void {
-    for (const tab of this._tabs) {
+    for (const tab of this.tabLabels) {
```

Comment:
To avoid mixin tab and TabLabels I recommend to use tabLabel here?
```suggestion
    for (const tabLabel of this.tabLabels) {
```